src/pages/[platform]/build-a-backend/storage/remove-files/index.mdx

```diff
@@ -0,0 +1,49 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Remove files',
+  description: 'Remove files using Amplify Storage',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+<BlockSwitcher>
+<Block name="API Usage">
```

Comment:
Non blocking comment: From a content perspective, would this be easier to read if we just inline both of the tabs (so no blockswitcher)? That way readers can also link to the section through the url. At the moment block switchers aren't linkable

Reply:
Good call, I will update this

Reply:
For this particular use case I think a block switcher is fine because they are immediately presented with the API usage when they access this link. I think it shows developers different ways of achieving it

Reply:
We are trying to move away from using block switchers because they have a number of drawbacks including:
1. Customers can't link to a particular way of doing things because it's hidden inside the block switcher tabs
2. Options aren't shown in the Table of Content (@timngyn to confirm if this is still the case)

@timngyn has done further research on this

Reply:
Yea, in this case since we don't have any other headings, this page won't have a table of contents. The "Amplify Console" tab won't be linkable as a result

![image](https://github.com/aws-amplify/docs/assets/54393192/dd1f9b7b-bcb1-4e8c-ad26-17c0c15325d0)

Reply:
I think that's not an issue for this particular case since it's a short and straightforward example and the block tabs itself act like the table of contents. I'd argue that it's visually more effective than the right-side table of contents. 

If there was a lot more content, I agree that it would be harder to find nested sections, there are none in this case.